File: CaesarCipher/JavaScript/PasswordCipher.js
function encrypt(text) {
        
    let result = "";
        
    for(let count = 0; count < text.length; count++) {
        let letter = text.charAt(count);
            
        let scrambledCharCode = letter.charCodeAt(0) + 3;
        
        let scrambledLetter = String.fromCharCode(scrambledCharCode);
            
        result += scrambledLetter;
    }
    return result;
}
 
    
function decrypt(text) {

    let result = "";
        
    for(let count = 0; count < text.length; count++) {
        let letter = text.charAt(count);
            
        let originalCharCode = letter.charCodeAt(0) - 3;
        
        let originalLetter = String.fromCharCode(originalCharCode);
            
        result += originalLetter;
    }
    return result;
}



const password = "Caeser";
console.log ("Original: " + password);
            
const encrypted = encrypt(password);
console.log("Encrypted: " + encrypted);
            
const decrypted = decrypt(encrypted);
console.log("Decrypted: " + decrypted);
    

File: CaesarCipher/Python/password_cipher.py
def encrypt(text):

    result = ""
    
    for letter in text:
    
        scrambled_letter = chr(ord(letter) + 3)
        
        result += scrambled_letter
        
    return result
    
    
def decrypt(text):

    result = ""
    
    for letter in text:
    
        original_letter = chr(ord(letter) - 3)
        
        result += original_letter
        
    return result
    
    
password = "Caeser"
print("Original: ", password)

encrypted = encrypt(password)
print("Encrypted: ", encrypted)

decrypted = decrypt(encrypted)
print("Decrypted: ", decrypted)
